Update fill-nulls-advanced.md (#5164)
dbt-labs/docs.getdbt.com: website/docs/docs/build/fill-nulls-advanced.md

| 2024-01-02 | 0 | 0 |
| 2024-01-03 |  8 | .1 |

Now, if you combine the metrics in a `derived` metric, there will be a zero value for `leads_to_website_visit` on `2024-01-02` and the final result set will not have any null values.

## FAQs

> <expandable alt_header="How to handle null values in derived metrics defined on top of multiple tables"/>

Reviewer: ```suggestion

<expandable alt_header="How to handle null values in derived metrics defined on top of multiple tables"/>
```

| 2024-01-02 | 0 | 0 |
| 2024-01-03 |  8 | .1 |

Now, if you combine the metrics in a `derived` metric, there will be a zero value for `leads_to_website_visit` on `2024-01-02` and the final result set will not have any null values.

## FAQs

<expandable alt_header="How to handle null values in derived metrics defined on top of multiple tables">

> For additional examples and discussion on how to handle null values in derived metrics that use data from multiple tables, check out [Metricflow issue #1031](https://github.com/dbt-labs/metricflow/issues/1031).

Reviewer: ```suggestion
For additional examples and discussion on how to handle null values in derived metrics that use data from multiple tables, check out [MetricFlow issue #1031](https://github.com/dbt-labs/metricflow/issues/1031).
```

| 2024-01-03 |  8 | .1 |

> Now, if you combine the metrics in a `derived` metric, there will be a zero value for `leads_to_website_visit` on `2024-01-02` and the final result set will not have any null values

Reviewer: ```suggestion
Now, if you combine the metrics in a `derived` metric, there will be a zero value for `leads_to_website_visit` on `2024-01-02` and the final result set will not have any null values.
```

| 2024-01-03 |  8 | .1 |

Now, if you combine the metrics in a `derived` metric, there will be a zero value for `leads_to_website_visit` on `2024-01-02` and the final result set will not have any null values

> ### Handling null values in derived metrics defined on top of multiple tables

Reviewer: hey @siljamardla , thanks so much for opening this and for providing such rich info! i think it makes sense to have this on the product docs, however i think placing this under an 'faqs' header will allow us the flexibility of adding more examples for diff use cases (rather than adding headers for specific use cases). we plan on having some more guides that detail specific use cases (cc @Jstein77 ) !  

let me know if you have strong opinions on this and i'm happy to collaborate further on this! 

## FAQs
<expandable alt_header="How to handle null values in derived metrics defined on top of multiple tables"/>

For additional examples and discussion on how to handle null values in derived metrics that use data from multiple tables, check out [Metricflow issue #1031](https://github.com/dbt-labs/metricflow/issues/1031).

</expandable>

> ### Handling null values in derived metrics defined on top of multiple tables

Reviewer: hey @siljamardla , thanks so much for opening this and for providing such rich info! i think it makes sense to have this on the product docs, however i think placing this under an 'faqs' header will allow us the flexibility of adding more examples for diff use cases (rather than adding headers for specific use cases). we plan on having some more guides that detail specific use cases (cc @Jstein77 ) !  

let me know if you have strong opinions on this and i'm happy to collaborate further on this! 
```suggestion
## FAQs
<expandable alt_header="How to handle null values in derived metrics defined on top of multiple tables"/>

For additional examples and discussion on how to handle null values in derived metrics that use data from multiple tables, check out [Metricflow issue #1031](https://github.com/dbt-labs/metricflow/issues/1031).

</expandable>
```

> See [Metricflow Issue #1031](https://github.com/dbt-labs/metricflow/issues/1031) for additional examples and discussion on how to handle null values in derived metrics that use data from multiple tables. 

Reviewer: ```suggestion
```
